packages/core/src/implementations/SnickerdoodleCore.ts

```diff
@@ -942,7 +956,10 @@ export class SnickerdoodleCore implements ISnickerdoodleCore {
         // Now the actual initialization!
         return accountService.initialize();
       })
-      .map(() => {});
+      .map(() => {
+        membershipWrapper.preFetch();
```

Comment:
make sure to add a .mapErr((e) => {logUtils.error(e);} on these, otherwise errors float away into the aether